client/dashboard/src/components/sources/Sources.tsx

```diff
@@ -99,27 +99,31 @@ export default function Sources() {
       return [];
     }
 
-    const openApiSources = deployment.openapiv3Assets.map((deploymentAsset) => {
-      const asset = assets.assets.find((a) => a.id === deploymentAsset.assetId);
-      if (!asset) {
-        throw new Error(`Asset ${deploymentAsset.assetId} not found`);
-      }
-      return {
-        ...asset,
-        deploymentAssetId: deploymentAsset.id,
-        name: deploymentAsset.name,
-        slug: deploymentAsset.slug,
-        type: "openapi" as const,
-      };
-    });
-
-    const functionSources = (deployment.functionsAssets ?? []).map(
-      (deploymentAsset) => {
+    const openApiSources = deployment.openapiv3Assets
+      .map((deploymentAsset) => {
+        const asset = assets.assets.find((a) => a.id === deploymentAsset.assetId);
+        if (!asset) {
+          console.error(`Asset ${deploymentAsset.assetId} not found`);
```

Comment:
Is it clear why this is firing in prod right now? This one seems like it's actually kinda unexpected behavior we should solve for? Is it like a pagination issue or something?

Reply:
Good question